SignalServiceKit/src/Messages/DeviceSyncing/OWSOutgoingSentMessageTranscript.m

```diff
@@ -50,10 +55,8 @@ - (OWSSignalServiceProtosSyncMessageBuilder *)syncMessageBuilder
     OWSSignalServiceProtosSyncMessageSentBuilder *sentBuilder = [OWSSignalServiceProtosSyncMessageSentBuilder new];
     [sentBuilder setTimestamp:self.message.timestamp];
 
-    // Sync messages have no thread or destination.
-    OWSAssert(!self.message.thread.contactIdentifier);
-    [sentBuilder setDestination:nil];
-    [sentBuilder setMessage:[self.message buildDataMessage:nil]];
+    [sentBuilder setDestination:self.sentRecipientId];
+    [sentBuilder setMessage:[self.message buildDataMessage:self.sentRecipientId]];
```

Comment:
This PR fixes two issues.  First "sent transcripts for outgoing messages in 1:1 messages" were broken. We need to include the "sent recipient id" in these transcripts.